test/nodejs/match.js

```diff
@@ -1,12 +1,14 @@
 var OSRM = require('../../');
 var test = require('tape');
-var berlin_path = require('./osrm-data-path').data_path;
+var data_path = require('./osrm-data-path').data_path;
+var kLocations = require('./locations').locations;
```

Comment:
Instead of `kLocations` and `kRoute` I would suggest using more explicit names like:

- `two_test_coordinates`
- `three_test_coordinates`